src/Constants.ts

```diff
@@ -28,7 +28,7 @@ export enum ENGINE {
 	ESLINT_CUSTOM = 'eslint-custom',
 	RETIRE_JS = 'retire-js',
 	CPD = 'cpd',
-	SFGE = 'sfge'
+	SFGE = 'Salesforce Graph Engine'
```

Comment:
Changed the name of the engine, which provided all the desired downstream effects.
Only potentially unintended side-effect is that `scanner:rule:list` now expects "Salesforce Graph Engine" instead of "sfge".

Reply:
Wouldn't this impact users automation if they are already using "sfge" in their scripts? I'm guessing this is also going to ripple effect into config.
Can we keep just the display text changes to "Salesforce Graph Engine" and not let it flow deeper for a minor release?

Reply:
I doubt customers are using `scanner:rule:list` in automated scripts, but if any are, it would affect them.
However, having done more testing, you're absolutely right that it cascades into configs. So yeah, I'll see if I can override just those two display strings. Stand by.